packages/react-native-codegen/src/parsers/flow/components/extends.js

```diff
@@ -10,10 +10,9 @@
 
 'use strict';
 
-import type {ExtendsPropsShape} from '../../../CodegenSchema.js';
 import type {TypeDeclarationMap} from '../../utils';
 
-function extendsForProp(prop: PropsAST, types: TypeDeclarationMap) {
```

Comment:
I won't refine this definition, unles it is crucial.

Also, if we have to, we can actually properly type this, as the final type is hardcoded to:
```ts
type extendsForProps = null | {
        type: 'ReactNativeBuiltInType',
        knownTypeName: 'ReactNativeCoreViewProps',
      }
```

Reply:
Sure. Was going to type the same but, but thought `null` mixin type seem odd, as passing `null` around is is bad ;)
Will see if there is need.